app/src/main/java/com/android/sample/MainActivity.kt

```diff
@@ -91,8 +99,31 @@ fun PlateSwipeApp() {
         startDestination = Screen.CREATE_RECIPE,
         route = Route.CREATE_RECIPE,
     ) {
-      composable(Screen.CREATE_RECIPE) { CreateRecipeScreen(navigationActions) }
+      composable(Screen.CREATE_RECIPE) { backStackEntry ->
+        remember(backStackEntry) { navController.getBackStackEntry(Route.CREATE_RECIPE) }
+
+        CreateRecipeScreen(
+            navigationActions = navigationActions, createRecipeViewModel = createRecipeViewModel)
+      }
+      composable(Screen.CREATE_RECIPE_INGREDIENTS) {
+        RecipeIngredientsScreen(
+            navigationActions = navigationActions,
+            createRecipeViewModel = createRecipeViewModel,
+            currentStep = 1)
```

Comment:
Great job on the navigation implementation! Could you please consider using a constant for `currentStep` to make the code a bit easier to maintain?  

Also, I noticed this section: `backStackEntry -> remember(backStackEntry) { navController.getBackStackEntry(Route.CREATE_RECIPE) }`. I’m curious why `CreateRecipeScreen` is called this way, rather than directly like the other screens. If there's a specific reason for this approach, a bit of documentation would really help clarify why this screen requires a unique setup. This context would be super valuable for anyone working on navigation in the future. Thanks!

Reply:
What do you mean by a  constant here. Do you want mean to define a default value in `RecipeIngredientsScreen` for currentStep for example. Or do you want me to write a constant in `C.kt` and use it in MainActivity?

Reply:
Write in `C.kt`, `CreateRecipeNameStep = 1` or something like that